docs/app/experiments/slider-marks.tsx

```diff
@@ -1,7 +1,7 @@
 'use client';
 import * as React from 'react';
 import * as Slider from '@base_ui/react/Slider';
-import { useSliderContext } from '@base_ui/react/Slider';
+import { useSliderContext } from '../../../packages/mui-base/src/Slider/Root/SliderContext';
```

Comment:
If there is a legitimate use case for the context, we should export it from the package.

Reply:
We discussed this and decided to revisit exporting Context(s) after the alpha phase